File: 프로그래머스/unrated/181927. 마지막 두 원소/마지막 두 원소.js
function solution(num_list) {
   let arr = num_list

    if(arr.at(-1) > arr.at(-2)) {
        arr.push(arr.at(-1) - arr.at(-2))
    } else {
        arr.push(arr.at(-1) * 2)
    }
    return arr
}

File: 프로그래머스/0/181927. 마지막 두 원소/마지막 두 원소.java
import java.util.Arrays;

class Solution {
  public int[] solution(int[] num_list) {
    // num_list 배열보다 하나 더 큰 arr 배열 복사하기
    int[] arr = Arrays.copyOfRange(num_list, 0, num_list.length+1);
    // num_list 마지막 인덱스
    int a = num_list[num_list.length-1];
    // num_list 그전 인덱스
    int b =  num_list[num_list.length -2];
    // 삼항 연산자
    int result = a > b ? a -b : a *2;
    // 마지막 자리에 값 추가
    arr[arr.length -1] = result;

    return arr;
  }
}